NetworkingServiceKit/Classes/Networking/AlamoNetworkManager.swift

```diff
@@ -193,29 +191,29 @@ open class AlamoNetworkManager: NetworkManager {
     ///
     /// - Parameter data: data stream
     /// - Returns: a response, empty dictionary if there were issues parsing the data
-    private func errorResponse(fromData data: Data?, request: URLRequest?) -> MSErrorDetails? {
-        var errorResponse: MSErrorDetails? = nil
+    private func errorDetails(fromData data: Data?, request: URLRequest?, code: Int, error: Error?) -> MSErrorDetails {
```

Comment:
Food for thought: Could this be a MSErrorDetails init() ? also should we use NSError.errorMessageKey instead of "message" ?